.dockerignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
+node_modules
+/dist
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw*
```

Comment:
this is equivalent to the `.gitnore`, can we be smarter here? This may very well qualify as premature optimisation and there's probably no need as we will likely hardly touch these files, but just wondered seeing this.

Reply:
Yeah, I would iterate on this. I'll add an issue.